crates/biome_cli/src/execute/std_in.rs

```diff
@@ -206,19 +178,8 @@ pub(crate) fn run<'a>(
                 Some("svelte") => SvelteFileHandler::output(&new_content, code.as_str()),
                 _ => code,
             };
-            if mode.is_check_apply() || mode.is_check_apply_unsafe() {
-                if output != new_content {
-                    new_content = Cow::Owned(output);
-                }
-            } else {
-                let diagnostic = FormatDiffDiagnostic {
-                    file_name: biome_path.display().to_string(),
-                    diff: ContentDiffAdvice {
-                        new: output,
-                        old: content.to_string(),
-                    },
-                };
-                diagnostics.push(Error::from(diagnostic));
+            if mode.is_check_apply() || mode.is_check_apply_unsafe() && output != new_content {
```

Comment:
This is just a confirmation. Is the order of evaluation of the conditions the same as below?


```
if (mode.is_check_apply() || mode.is_check_apply_unsafe()) && output != new_content {
    new_content = Cow::Owned(output);
}
```

Reply:
Good catch!